_tests/browser_tests/page_objects/page_the-bureau.js

```diff
@@ -0,0 +1,10 @@
+var TheBureauPage = function() {
+  this.get = function() {
+    browser.get( '/the-bureau' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle() };
```

Comment:
It appears this can't be moved, everything I try I get an error. Will explore again later.